services/venice-controller/src/main/java/com/linkedin/venice/controller/ErrorPartitionResetTask.java

```diff
@@ -110,11 +111,15 @@ public void run() {
   }
 
   private void resetApplicableErrorPartitions(Store store) {
-    int currentVersion = store.getCurrentVersion();
-    if (currentVersion == Store.NON_EXISTING_VERSION || !store.getVersion(currentVersion).isPresent()) {
+    int currentVersionNumber = store.getCurrentVersion();
```

Comment:
Ok.  I 'think' this shouldn't be the case, but is there a chance that there isn't a current version?  I think even in the presence of no version we default the version number to 0

Reply:
ah wait nm.  Doesn't matter, the check below covers this.